core/src/main/kotlin/com/katanox/tabour/configuration/core/Core.kt

```diff
@@ -0,0 +1,10 @@
+package com.katanox.tabour.configuration.core
+
+import com.katanox.tabour.Tabour
+import com.katanox.tabour.consumption.Config
+
+/** Creates a new [Tabour] instance */
+fun tabour(init: Tabour.Configuration.() -> Unit): Tabour =
```

Comment:
Question:  Will we allow the creation of multiple tabor instances

Reply:
yes but you dont need to do that since it does not achieve any goal

Reply:
then i would suggest that we restrict the number of instances and make sure it is a singlton